WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -45,6 +45,17 @@ final class OrderListViewController: UIViewController, GhostableViewController {
         return dataSource
     }()
 
+    /// Returns the first Order in the OrderList datasource
+    ///
+    var firstOrderInIndexPath: Order? {
```

Comment:
Opinion: I don't think adding `IndexPath` is necessary. Maybe rename this into `firstAvailableOrder`. Please correct me if I am missing something.

Reply:
Agree completely, changed on https://github.com/woocommerce/woocommerce-ios/pull/11571/commits/1a2b5b6cc9f6231048a03d00b31185ea6ff82d7f